### PR TITLE
fix(windows): re-enable signature check 🔥

### LINKED ISSUE #9692: bug(developer,windows): 'Unknown Publisher' message displayed on the screen during the installation
### Describe the bug

I observed that a warning message 'Unknown Publisher'  is shown on the screen when installing the Keyman Developer 17.0.186 build.

### Reproduce the bug

1. Download Keyman 17.0.186-alpha build.
2. Double-click on the Keyman 17.0.186-alpha.exe file. 

Here, I noticed that a warning message appears on the screen with the 'Run anyway' and 'Don't run' buttons. 

Note: I was able to reproduce this issue on Windows 11 OS (VM).

I have attached the Screenshots for reference. 

..Warning message in Windows 10 OS

   

..Warning message in Windows 11 OS

   

### Expected behavior

'Unknown Publisher' message should not appear on the screen during Keyman Developer installation in Windows OS. 

### Related issues

_No response_

### Keyman apps

- [ ] Keyman for Android
- [ ] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [ ] Keyman for macOS
- [ ] Keyman for Windows
- [X] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

### Keyman version

17.0.186-alpha

### Operating system

Windows 10 OS, Windows 11 OS (VM)

### Device

_No response_

### Target application

_No response_

### Browser

_No response_

### Keyboard name

_No response_

### Keyboard version

_No response_

### Language name

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by mcdurdin**
Fixes #9692.

Signature checking was skipped because we missed a ".virtual" to force nmake to build the test and test_i3633 targets. This opened up a small cascade of related formatting issues on Makefiles, and the fact that the test_i3633 (has there ever been a more poorly named project?) Makefile did not even work.

Refactored significantly, added same tests to Developer Makefile, and also now verifying the .msi and installer executable.

We can improve this further but I'd like to get this in to avoid further critical issues with code signing given the current broken signing configuration.

@keymanapp-test-bot skip

### CODE REVIEW

File: windows/src/test/verify_signatures/Makefile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+!include ..\..\Defines.mak
+
+test: prereq
+  $(SIGCHECK) $(ROOT)\bin\desktop\* > sig1
+  $(SIGCHECK) $(ROOT)\bin\engine\* >> sig1
+  $(SIGCHECK) $(ROOT)\bin\inst\* >> sig1
+  $(VERIFY_SIGNATURES) < sig1
+
+# prereq may not be needed?
```
Comment (rc-swag) [inline windows/src/test/verify_signatures/Makefile:13]: Why this comment?  I think the prereq is needed to build the verify_signatures binary

**Review by rc-swag — APPROVED** [review]
LGTM - just the one comment about a comment in the makefiles.

File: windows/src/test/verify_signatures/Makefile (hunk 1 above)
Comment (mcdurdin) [reply windows/src/test/verify_signatures/Makefile:13]: The prereq may already have been built, but it's not a big deal because it builds quickly anyway.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.187-alpha](https://keyman.com/downloads/releases/alpha/17.0.187)

**Pull request merged**